cfme/tests/v2v/test_v2v_ansible.py

```diff
@@ -117,62 +133,33 @@ def test_migration_playbooks(request, appliance, v2v_providers, host_creds, conv
         request, appliance, credential.name, ansible_repository, "retire"
     )
 
-    infrastructure_mapping_collection = appliance.collections.v2v_mappings
-    mapping = infrastructure_mapping_collection.create(
-        form_data_vm_obj_single_datastore.form_data
-    )
+    infrastructure_mapping_collection = appliance.collections.v2v_infra_mappings
+    mapping_data = mapping_data_vm_obj_single_datastore.infra_mapping_data
+    mapping = infrastructure_mapping_collection.create(**mapping_data)
 
     @request.addfinalizer
     def _cleanup():
         infrastructure_mapping_collection.delete(mapping)
 
     # vm_obj is a list, with only 1 VM object, hence [0]
-    src_vm_obj = form_data_vm_obj_single_datastore.vm_list[0]
+    src_vm_obj = mapping_data_vm_obj_single_datastore.vm_list[0]
 
-    migration_plan_collection = appliance.collections.v2v_plans
+    migration_plan_collection = appliance.collections.v2v_migration_plans
     migration_plan = migration_plan_collection.create(
         name="plan_{}".format(fauxfactory.gen_alphanumeric()),
         description="desc_{}".format(fauxfactory.gen_alphanumeric()),
         infra_map=mapping.name,
-        vm_list=form_data_vm_obj_single_datastore.vm_list,
-        start_migration=True,
+        vm_list=mapping_data_vm_obj_single_datastore.vm_list,
+        target_provider=provider,
         pre_playbook=provision_catalog.name,
         post_playbook=retire_catalog.name,
+        pre_checkbox=True,
+        post_checkbox=True
     )
+    assert migration_plan.plan_started
```

Comment:
Do these need wait_for or anything? Does it instantly go through all these states? 

Just some in-line comments here explaining the assumptions and why these states can be immediately asserted against, if there's no chance of a race condition causing test failure.